<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BTreeWindow</class>
 <widget class="QMainWindow" name="BTreeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>B-树</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/BTreeICO.png</normaloff>:/images/images/BTreeICO.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>351</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxNode">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>110</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>结点操作</string>
    </property>
    <widget class="QLineEdit" name="lineKey">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>26</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>输入整数键值</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsert">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>58</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>插  入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>84</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>删  除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>查  找</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnReturn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>返  回</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxInfor">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>10</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>当前信息</string>
    </property>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>23</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>·树  阶：</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDegree">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>23</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelDepth">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>44</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>44</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>·深  度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>65</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>·键值数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNum">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>66</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSet">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>260</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>重设阶数</string>
    </property>
    <widget class="QLineEdit" name="lineDegree">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>26</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>输入整数阶值</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReset">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>52</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确  认</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxTree">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>350</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>树操作</string>
    </property>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>110</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>清  空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>82</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>收  起</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnShow">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>55</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>展  开</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>28</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>显示失误结点</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
